main/lsp/requests/completion.cc

```diff
@@ -58,6 +58,7 @@ const RubyKeyword rubyKeywords[] = {
     {"end",
      "The end of a syntax block. Used by classes, modules, methods, exception handling and control expressions."},
     {"ensure", "Starts a section of code that is always run when an exception is raised."},
+    {"enum", "Creates an enum class", "class $1 < T::Enum\nenums do\n$0\nend\nend"},
```

Comment:
I believe you still want the indentation, like `\n  enums`. If the tests passed without that, I'd be a little worried 🤔

Reply:
Fixed it! Forgot a `:` in `apply-completion` while writing the test turns out :3. Hence, just about anything was passing.